src/it/scala/io/iohk/ethereum/transactionTest/ContractTest.scala

```diff
@@ -0,0 +1,47 @@
+package io.iohk.ethereum.transactionTest
```

Comment:
Let's stick to [naming conventions](http://stackoverflow.com/questions/12534302/why-should-java-package-name-be-lowercase)

Also could the name refer to execution? Maybe `TxExecTest` / `txexectest`? Those test are about transaction execution, not transaction signing for example.